pwa/app/components/tba/allianceSelectionTable.tsx

```diff
@@ -80,7 +81,7 @@ export default function AllianceSelectionTable(props: {
         <TableBody>
           {props.alliances.map((a, idx) => (
             <AllianceTableRow
-              key={a.name}
+              key={a.name ?? `alliance-${idx}`}
```

Comment:
Should we just make the key the index? i don't think the test for name and fallback to index is necessary.